libs/ngx-mime/src/lib/core/alto-service/html.formatter.ts

```diff
@@ -34,13 +35,38 @@ export class HtmlFormatter {
         styles.push('font-weight: bold');
       }
 
+      var paragraphs = words.join(' ');
+
+      if(this.searchQuery || this.hits){
+        paragraphs = this.transform(paragraphs, this.searchQuery);
+      }
+
       html += '<p';
       if (styles && styles.length > 0) {
         html += ` style="${styles.join(';')}"`;
       }
-      html += `>${words.join(' ')}<p/>`;
+      html += `>${paragraphs}<p/>`;
     });
 
+
     return this.sanitizer.bypassSecurityTrustHtml(html);
   }
+
+  transform(html: any, searchQuery: any): any {
```

Comment:
1. Don't use `any`. Both parameters and function return type can be typed.
2. Is `searchQuery` needed as input param? Can you use `this.searchQuery` inside the function instead?
3. is the name `html` correct here? The code that uses this function pass a variable with the name `paragraphs`. Should they match each other?